phpstan.neon.dist

```diff
@@ -1,9 +1,10 @@
 parameters:
-  level: 6
+  level: 7
   paths:
     - lib
 
   ignoreErrors:
+    - '#Method Buzz\Client\MultiCurl::sendRequest() should return Psr\Http\Message\ResponseInterface but returns Psr\Http\Message\ResponseInterface|null#'
```

Comment:
Be careful here, this is a regexp, you need to escape the \, the (, the ) and also |.

Reply:
Yeah, I know it not perfect. I did work so I was happy with it =)

Im happy to review a fix.